Cargo.toml

```diff
@@ -128,6 +128,7 @@ shiplift = { git = "https://github.com/LucioFranco/shiplift", branch = "timber"
 owning_ref = "0.4.0"
 listenfd = "0.3.3"
 inventory = "0.1"
+strip-ansi-escapes = "0.1.0"
```

Comment:
Not sure if we want to add this dependency. It adds a minimal amount of extra dependencies, and it gets the job done.

Alternatively, we could do the escape sequence parsing ourselves, but it's a bit more complex than writing a simple regular expression, so we'd probably want to write a tiny parser if we go that route.

Another option would be to simplify the implementation and only account for the most common ["CSI" escape sequence](https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_sequences) patterns.

Reply:
This definitely seems worthwhile to me, especially as a pretty light dependency.